test/libsolidity/semanticTests/various/selfdestruct_post_cancun_multiple_beneficiaries.sol

```diff
@@ -0,0 +1,60 @@
+contract C {
+    constructor() payable {}
+    function terminate(address _beneficiary) public {
+        selfdestruct(payable(_beneficiary));
+        assert(false);
+    }
+}
+
+contract D {
+    address account1 = payable(0x5e17b14ADd6c386305A32928F985b29bbA34Eff5);
+    address account2 = payable(0x9d83e140330758a8fFD07F8Bd73e86ebcA8a5692);
```

Comment:
You can use any address here, right? You could use some more recognizable ones :)
```suggestion
    address account1 = payable(0x1111111111111111111111111111111111111111);
    address account2 = payable(0x2222222222222222222222222222222222222222);
```

Reply:
Haha yes indeed.